src/code/z_kankyo.c

```diff
@@ -252,7 +252,7 @@ void Environment_Init(PlayState* play2, EnvironmentContext* envCtx, s32 unused)
 
     gSaveContext.sunsSongState = SUNSSONG_INACTIVE;
 
-    if (((void)0, gSaveContext.save.dayTime) > CLOCK_TIME(18, 0) ||
+    if (((void)0, gSaveContext.save.dayTime) > (0, CLOCK_TIME(18, 0)) ||
         ((void)0, gSaveContext.save.dayTime) < CLOCK_TIME(6, 30)) {
```

Comment:
should this be (void)0? whats happenin here

Reply:
It's a fake match from the permuter, just forgot to put the void there

Reply:
ok, maybe we should label it and revisit at some point.

I encountered a case recently where putting a void0 on a save context member matched retail, but broke debug. but then putting it on *another* access fixed both retail and debug. point being they seem really finnicky and we need to one day figure out which ones are real macros 😄

Reply:
Fair, yeah it's probably something like that. It's very finnicky. Added a fake label